src/components/Common/SubtitleWithBody.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Subtitle, BodyPrimary } from '../Typography'
+
+const ItemBody = styled(BodyPrimary).attrs({
+  noPaddingTop: true,
+  muted: props => props.bg === 'dark',
+  reverse: props => props.bg === 'dark'
+})`
+  > a {
+    text-decoration: underline;
+  }
+`
+
+const ItemSubtitle = styled(Subtitle).attrs({
+  reverse: props => props.bg === 'dark'
+})`
+  padding-bottom: 0;
+`
+
+const SubtitleWithBody = ({ subtitle, body, bg = 'white' }) => (
+  <React.Fragment>
```

Comment:
This really bothers me :(

Just import fragment lazy ahah

Reply:
ok!

Reply:
`React.Lazy` ! 😄 
done!